tests/permissions/test_set_permission.py

```diff
@@ -39,48 +39,47 @@ async def test_permissions_set_permission(websocket, context_id, example_url):
 @pytest.mark.asyncio
 @pytest.mark.skip(reason="See chromium-bidi/issues#1610")
 async def test_permissions_set_permission_in_user_context(
-        websocket, context_id, example_url):
+        websocket, context_id, example_url, create_context):
     await goto_url(websocket, context_id, example_url)
 
     user_context = await execute_command(websocket, {
         "method": "browser.createUserContext",
         "params": {}
     })
 
-    browsing_context = await execute_command(
-        websocket, {
-            "method": "browsingContext.create",
-            "params": {
-                "type": "tab",
-                "userContext": user_context["userContext"]
-            }
-        })
-
+    another_browsing_context = await create_context(
+        user_context_id=user_context["userContext"])
```

Comment:
I don't think we need anything except browsing context id, so something like this would be even more readable:
```suggestion
    another_browsing_context_id = (await create_context(
        user_context_id=user_context["userContext"]))['context']
```

Reply:
Why does create_context already return an ID?